experimental/examples/attributable-map-demo/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This should extend our global config (prob just copy the tsconfig from another demo)